python/lsst/summit/extras/logUtils.py

```diff
@@ -82,7 +88,7 @@ def _getDataRefs(self):
         dataRefs : `list` [`lsst.daf.butler.core.datasets.ref.DatasetRef`]
         """
         results = self.butler.registry.queryDatasets(
-            f"{self.taskName}_log", collections=self.collection, findFirst=True
+            f"{self.taskName}_log", collections=self.collection, findFirst=True, where=self.whereStr
```

Comment:
I assume the default/null option here is an empty string and not `None`, right?

Reply:
Yup, it seems to be `""` for `where` and `None` for `bind`:
https://github.com/lsst/daf_butler/blob/main/python/lsst/daf/butler/registry/_registry.py#L1103-L1116